src/server/src/main/java/io/cassandrareaper/ReaperApplicationConfiguration.java

```diff
@@ -65,6 +65,11 @@
   @DefaultValue("false")
   private Boolean incrementalRepair;
 
+  @JsonProperty
+  @NotNull
+  @DefaultValue("false")
+  private Boolean blacklistTwcsTables;
```

Comment:
The name of that variable suggests that we're blacklisting TWCS tables from all repair runs, which isn't exactly what it does. Currently we only blacklist for scheduled runs, not manual ones.
We could either:
- rename it to `blacklistTwcsTablesForScheduledRuns`
or
- blacklist TWCS/DTCS tables in manual runs as well.

Reply:
Blacklisting now occurs when repairs are triggered, regardless of whether they were automatic, scheduled, or manual.